lib/ansible/modules/cloud/amazon/iam_server_certificate_facts.py

```diff
@@ -1,10 +1,7 @@
 #!/usr/bin/python
-# Copyright: Ansible Project
-# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
-
-from __future__ import absolute_import, division, print_function
```

Comment:
We can' t remove these `__future__` imports, they're important for python 2/3 compatibility.